apps/system/js/app_usage_metrics.js

```diff
@@ -705,46 +723,50 @@
     // lockscreen right before sleeping, for example.)
     time = Math.round(time / 1000);
     if (time > 0) {
-      var usage = this.getAppUsage(app);
+      var usage = this.getAppUsage(app.manifestURL);
       usage.invocations++;
       usage.usageTime += time;
       this.needsSave = true;
       debug(app, 'ran for', time);
     }
+    return time > 0;
```

Comment:
I don't see you using these boolean return values anywhere. Are you planning to use them in a future patch? Seems unnecessary, but harmless.